### ISSUE TITLE
Question: Is there an example for YTDL_OPTIONS with "match_filter"?

### ISSUE THREAD

**Issue by cringe-alphabrik** [alexta69/metube#229]
This is not so much an issue but a question: I want to filter out youtube shorts which I previously did with `yt-dlp --match-filter="duration>61"`. I tried to configure the env variable `YTDL_OPTIONS={"match_filter":"duration>61"}` which results in an error on download, because `match_filter` expects a function instead of the string option.

I tried to read [`match_filter` in `utils.py`](https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/utils.py#L3912) but I can't figure out how to configure filters. Can anyone point me in the right direction or has an example for filtering videos?

**Comment by PikuZheng** [comment]
[Filter videos](https://github.com/yt-dlp/yt-dlp/blob/40d77d89027cd0e0ce31d22aec81db3e1d433900/README.md?plain=1#L1949) fyi.
yes, it looks like it wants a function. maybe you need to write a little code and recompile the container

**Comment by alexta69** [comment]
I don't think we can do anything with this now, please reopen if you think otherwise
